### ISSUE TITLE
[BUG]: Pressing ENTER editing any text input field of Rally Item actions first available RVI button

### ISSUE THREAD

**Issue by AdamPiotrowski91** [AdamPiotrowski91/RVI_Ideas#91]
**Description**
As title says - for some reason pressing enter during edition actions first or last RVI added button.

**Steps To Reproduce**

1. Open any item in Rally in fullscreen mode or in side panel
2. Start editing it's name or any other text input (don't have to change anything)
3. Press ENTER
4. Observe some RVI action happening

**Expected Behavior**
ENTER does not do anything, especially not unprompted

**Screenshots**

N/A

**Environment:** *(Leave only applicable items.)*

- *OS*:
  - Mac
  - Windows
  - Linux
- *Browser*:
  - *Type*:
    - Chrome

**Additional context**

Requestor: PO / me

**Comment by AdamPiotrowski91** [comment]
This was the reason: <https://stackoverflow.com/a/12914700>

TL;DR: `type="button"` because by default it's `submit` so ENTER submits...
